docs/changelog.rst

```diff
@@ -32,13 +32,14 @@ Release 1.5.0 (2019-02-01)
   `#634`)
```

Comment:
Is there an underscore missing after 634 ?

Reply:
Looks like it - feel free to push little fixes like that when you see them (if you have time).